problem13.rb

```diff
@@ -102,8 +103,7 @@
 72107838435069186155435662884062257473692284509516
 20849603980134001723930671666823555245252804609722
 53503534226472524250874054075591789781264330331690'.split
-result=0
-numbers.each do |i|
-    result+=i.to_i
-end 
+   
+result=numbers.inject{|i, sum| sum.to_i+i.to_i}
+ 
 puts (result.to_s)[0,10]  
```

Comment:
```suggestion
numbers.map(&:to_i).sum.to_s.first(10)
```
Немного подробнее:
- numbers это массив строк с числами, чтобы получить из него массив чисел, надо применить к каждому метод to_i. Для этого можно передать метод to_i как параметр в итератор, используя метод &. Подробнее [тут](https://www.honeybadger.io/blog/how-ruby-ampersand-colon-works/). Так же советую почитать про то, что такое блоки, проки и люмбды в руби (можно в книжке, можно просто поискать статьи, их много)
- дальше все просто: находим сумму элементов числового массива с помощью #sum, приводим к строке и берем первые 10 символов с помощью #first